### ISSUE TITLE
AI Tiered Plans: prepare UsagePanel to deal with usage above allowance

### ISSUE THREAD

**Issue by retrofox** [Automattic/jetpack#33859]
Address the case where the usage is above the allowance for any reason, resulting on weird displays like "41 / 20 free requests":

![Image](https://user-images.githubusercontent.com/6760046/278102878-ab0bb335-2157-4e32-8f02-20558c861b04.png)

- Should we truncate the value and show 20 / 20? The downside is the user seeing the real "41" after the upgrade and do not understand the suden bump
- Should we just keep it as is?

**Comment by retrofox** [comment]
> Address the case where the usage is above the allowance for any reason

Do you know if we can create the cases that generate this situation?

Should the progress bar be red?

> Should we just keep it as is?

Probably, yes. We can try clarify the state in the help section of the component

**Comment by lhkowalski** [comment]
> Do you know if we can create the cases that generate this situation?

I think it can happen for the 500 plan, since we will not have a hard limit for it, so the user may have "537 / 500 requests". But it's a case for few users.

> Should the progress bar be red?

I think so, yes, to make it clear that it's over the limit.

---

I think this is an edge case that we can wait to address in the future, and just let the number goes over the limit. I don't expect it to be common, even on the 500 requests plan, so it's low priority.

**Comment by lhkowalski** [comment]
After #34195, the usage bar now turns red when the usage is `>=` the licensed limit.

The usage number is showed regardless of being over the limit, resulting in cases like `1221 / 1000 requests`, but with the red bar denoting it's over the limit.

I think we can call this done. We can reopen or fill a new issue in case we want to change something.
